framework_tvb/tvb/core/services/import_service.py

```diff
@@ -409,6 +409,15 @@ def load_datatype_from_file(self, current_file, op_id, datatype_group=None, curr
         """
         self.logger.debug("Loading DataType from file: %s" % current_file)
         h5_class = H5File.h5_class_from_file(current_file)
+        reference_list = h5_class(current_file).gather_references()
+
+        for reference in reference_list:
```

Comment:
Access the tuple element you need directly in `for` instead of having to do `reference[1]` later :
`for _, reference_gid in reference_list`